app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditActivityPresenter.kt

```diff
@@ -16,6 +16,7 @@ class ProfileEditActivityPresenter @Inject constructor(
 
   private lateinit var toolbar: Toolbar
 
+  /** This handles the Intent creation of the [ProfileEditActivity]. */
```

Comment:
Shouldn't this be `Handles onCreate function of [ProfileEditActivity].`

Reply:
done